src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -174,6 +174,20 @@
     <PlatformManifestFileEntry Include="Microsoft.DiaSymReader.Native.amd64.dll" IsNative="true" FallbackFileVersion="$(MicrosoftDiaSymReaderNativeFileVersion)" />
     <PlatformManifestFileEntry Include="Microsoft.DiaSymReader.Native.arm.dll" IsNative="true" FallbackFileVersion="$(MicrosoftDiaSymReaderNativeFileVersion)" />
     <PlatformManifestFileEntry Include="Microsoft.DiaSymReader.Native.arm64.dll" IsNative="true" FallbackFileVersion="$(MicrosoftDiaSymReaderNativeFileVersion)" />
+    <!-- NativeAOT specific files -->
+    <PlatformManifestFileEntry Include="libbootstrapper.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="libbootstrapperdll.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="libeventpipe-disabled.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="libeventpipe-enabled.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="libnumasupportdynamic.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="libnumasupportstatic.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="libRuntime.ServerGC.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="libRuntime.WorkstationGC.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="libstdc++compat.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="System.Private.DisabledReflection.dll" />
+    <PlatformManifestFileEntry Include="System.Private.Reflection.Execution.dll" />
+    <PlatformManifestFileEntry Include="System.Private.StackTraceMetadata.dll" />
+    <PlatformManifestFileEntry Include="System.Private.TypeLoader.dll" />
```

Comment:
Why are these needed? I thought NativeAOT ships as native binary?
Is System.Private.Corelib.dll missing in the list?

Reply:
These are things that could be in CoreLib but since it doesn't cost anything to have them out of CoreLib, we keep them out of CoreLib. We folded some into CoreLib in the past. We definitely need CoreLib too and it's already include above.

Reply:
I see so there is a circular dependency between these and SPC?

Reply:
> Is System.Private.Corelib.dll missing in the list?

SPC is already included at line 74.